utilities/BuildUtilities.groovy

```diff
@@ -624,3 +635,48 @@ def assertDbbBuildToolkitVersion(String currentVersion){
 		System.exit(1)
 	}
 }
+
+/*
+ * Logs the resolution rules of the DependencyResolver in a table format
+ * 
+ */
+def printResolutionRules(List<ResolutionRule> rules) {
+
+	println("*** Configured resulution rules:")
+	
+	// Print header of table
+	println("    " + "Library".padRight(10) + "Category".padRight(12) + "SourceDir/File".padRight(50) + "Directory".padRight(36) + "Collection".padRight(24) + "Archive".padRight(20))
+	println("    " + " ".padLeft(10,"-") + " ".padLeft(12,"-") + " ".padLeft(50,"-") + " ".padLeft(36,"-") + " ".padLeft(24,"-") + " ".padLeft(20,"-"))
+
+	// iterate over rules configured for the dependencyResolver
+	rules.each{ rule ->
+		searchPaths = rule.getSearchPath()
+		searchPaths.each { DependencyPath searchPath ->
+			def libraryName = (rule.getLibrary() != null) ? rule.getLibrary().padRight(10) : "N/A".padRight(10)
+			def categoryName = (rule.getCategory() != null) ? rule.getCategory().padRight(12) : "N/A".padRight(12)
+			def srcDir = (searchPath.getSourceDir() != null) ? searchPath.getSourceDir().padRight(50) : "N/A".padRight(50)
+			def directory = (searchPath.getDirectory() != null) ? searchPath.getDirectory().padRight(36) : "N/A".padRight(36)
+			def collection = (searchPath.getCollection() != null) ? searchPath.getCollection().padRight(24) : "N/A".padRight(24)
+			def archiveFile = (searchPath.getArchive() != null) ? searchPath.getArchive().padRight(20) : "N/A".padRight(20)
+			println("    " + libraryName + categoryName + srcDir + directory + collection + archiveFile)
+
+		}
+	}
```

Comment:
I can't test (no time) but can you confirm the padding?
In the output I see in the conversation the Collection and archive file are not at the correct place.
![image](https://user-images.githubusercontent.com/60784175/153426662-9696d095-0a6b-413c-a759-f8dbc35e9a89.png)

Reply:
I have redone the padding - See the second comment ... https://github.com/IBM/dbb-zappbuild/pull/171#issuecomment-1033974162